package.json

```diff
@@ -1,21 +1,18 @@
 {
   "name": "mindart_pwa",
   "version": "1.2.17",
-  "description": "mind art progressive web app, for offline use",
+  "description": "mind art progressive web app",
   "scripts": {
-    "build": "esbuild --outfile=sw.js --bundle sw.ts && workbox injectManifest",
-    "serve": "dhost ."
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
   },
+  "type": "module",
   "dependencies": {
-    "@material/button": "^14.0.0",
-    "esbuild": "^0.18.2",
-    "material-components-web": "^14.0.0",
-    "material-ui": "^0.20.2",
-    "update": "^0.4.2",
-    "workbox-cli": "^7.0.0"
+    "p5": "^1.11.3"
   },
   "devDependencies": {
-    "esbuild": "^0.19.3",
-    "workbox-precaching": "^7.0.0"
+    "vite": "^6.1.0",
```

Comment:
local dev, production build and service worker creation are all handled by [vite](https://vite.dev/) now